pages/login/index.tsx

```diff
@@ -12,21 +12,18 @@ import {
   Button,
 } from '@chakra-ui/react';
 import { csrfToken, signIn, getSession } from 'next-auth/client';
-import { useRouter } from 'next/router';
 
 interface Props {
   token: string;
 }
 
 const Login: NextPage<Props> = ({ token }) => {
-  const { push } = useRouter();
   const { errors, touched, values, handleChange, handleSubmit } = useFormik({
     initialValues: {
       email: '',
     },
     onSubmit: async ({ email }) => {
-      await signIn('email', { email });
-      push('/');
```

Comment:
Currently in safari you can't login, it's because this `push` is re routing back to login so you never get to see the email sent screen. We don't need this, the callback will get us to the verify screen